### ISSUE TITLE
syllables not coming out correctly on the website

### ISSUE THREAD

**Issue by factormystic** [nlp-compromise/website#16]
right now, `houston texas` is
- houston
- texas

which is wrong. but `houstontexas` is
- hous
- ton
- te
- xas

and `houston` and `texas` separately are also correct. pretty sure this is a website thing, because:

```
let s = nlp.text("houston texas").syllables();
> [ [ [ 'hous', 'ton' ], [ 'te', 'xas' ] ] ]
```

**Comment by spencermountain** [comment]
yeah, thanks. this nested response format is pretty nutty. Term.syllables() should definitely always return an array of strings, no matter how many words are in the term. I don't know why it's doing that.

**Comment by nloveladyallen** [comment]
Huh. I'm seeing it one level deeper than @factormystic.

``` js
> nlp.text('houston texas').syllables()
[ [ [ [Object], [Object] ] ] ]
> nlp.text('houston texas').syllables()[0]
[ [ [ 'hous', 'ton' ], [ 'te', 'xas' ] ] ]
```

**Comment by spencermountain** [comment]
yeah. I also had the thought that maybe it should return `this`, just like all the other text/sentence methods do - that way you can be smart about whitespace and all.
